plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -42,14 +53,52 @@ public void execute(@CanManage User deputy, Guild guild, String[] args) {
         when(args.length < 1, this.messages.generalNoNickGiven);
         when(guild.getMembers().size() >= this.config.maxMembersInGuild, formatter.format(this.messages.inviteAmount));
 
-        User invitedUser = UserValidation.requireUserByName(args[0]);
+        if (args[0].equals(this.config.inviteCommandAllArgument)) {
+
+            Double range;
+
+            try {
+                range = Math.pow(Double.parseDouble(args[1]), 2);
+            }
+            catch (Exception exception) {
+                range = this.config.inviteCommandAllMaxRange;
+            }
+
+            Double finalRange = range;
```

Comment:
Brudny hack, zamiast usprawnić sekcje wyżej. To powinna być jedna linijka, max. 2-3 bez pustych przerw. Wystarczy naprawić, że config nie trzyma wartości podniesionej do potęgi i ogarnąć jakieś parsowanie z fallbackiem — pewnie jako metoda w utilsach, od biedy lokalny static. Coś w stylu `double range = Blabla.parseDouble(args[1], this.config.blabla)` (używanie `#distanceSquared` zbędne jak wspomniałem w innych komentarzach).